marklogic-data-hub/src/main/java/com/marklogic/hub/hubcentral/migration/FlowMigrator.java

```diff
@@ -149,6 +149,10 @@ public void migrateFlows() {
             ObjectNode newSteps = nodeFactory.objectNode();
             for (Map.Entry<String, Step> entry : steps.entrySet()) {
                 Step step = entry.getValue();
+                if (step.getStepId() != null) {
+                    newSteps.set(entry.getKey(), nodeFactory.objectNode().put("stepId", step.getStepId()));
```

Comment:
The flow document is migrated and hence steps will already have  "stepId" . So, do we have to set "stepId" again in the flow ?

Reply:
I believe so, since "newSteps" is assigned to the flow.

Reply:
But since the flow is already migrated, shouldn't it already have 'stepId' rather than inline steps ? so, we don't have to set 'stepId's in flow again right ?

Reply:
Yes, because it is building a new flow. You have to transfer the minimal {"1": "stepId"} to the new flow.